featurebyte/routes/catalog/api.py

```diff
@@ -66,6 +67,14 @@ def __init__(self) -> None:
             status_code=HTTPStatus.OK,
         )
 
+        # update online store route
+        self.router.add_api_route(
+            "/{catalog_id}/online_store_async",
```

Comment:
🤔 Our current async function does not have `async` in the endpoint.

Reply:
Understand this is for backward compatibility.. But it introduces some inconsistencies of existing routes naming convention.

Reply:
Sure, I'm open to suggestions.

Reply:
I plan to rename `online_store_async` back to `online_store` once UI transitioned, so this should be temporary.

Reply:
I think frontend is using status code to check whether it is an async task or not. Thinking whether it is ok to just replace existing sync patch endpoint.

Reply:
Good point, will check if we can simply replace the existing route.